### ISSUE TITLE
IE9/8 dropdown is wrong color??

### ISSUE THREAD

**Issue by meganem** [SU-SWS/stanford_light#53]
Cardinal:
![screen shot 2014-11-24 at 2 38 49 pm](https://cloud.githubusercontent.com/assets/1919260/5174612/a7f6e2c4-73e7-11e4-8242-e1112142ef91.png)
Should be:
![screen shot 2014-11-24 at 2 39 29 pm](https://cloud.githubusercontent.com/assets/1919260/5174617/b2557bb8-73e7-11e4-87fd-cd282da84db5.png)

also doing the same thing in dark:
![screen shot 2014-11-24 at 2 40 38 pm](https://cloud.githubusercontent.com/assets/1919260/5174639/e05f1ad2-73e7-11e4-8946-744d56bd2356.png)

and bright:
![screen shot 2014-11-24 at 2 41 19 pm](https://cloud.githubusercontent.com/assets/1919260/5174643/f6736922-73e7-11e4-97d0-0a75660fa3c8.png)

and probably others. Like that beige color is overriding the color set by the theme.

**Comment by hyperboy** [comment]
Migrated to https://github.com/SU-SWS/stanford_framework/issues/121
